### PR TITLE
Support sequences in pg_dump+dbzm live migration 

### PR DESCRIPTION
**Pull request by makalaaneesh**
- When starting dbzm for the cdc phase, extract sequence last values from pg_dump output and init dbzm with those values. This is needed to handle scenarios where there are no events corresponding to a table in the cdc phase (in which case debezium will not be able to track the max value of the corresponding sequences). 
- Support case sensitivity in sequence names

Caveat:
- sequences that are not owned by any column are not supported.

### CODE REVIEW

File: yb-voyager/cmd/exportData.go [hunk 1]
```diff
@@ -264,6 +277,39 @@ func exportPGSnapshotWithPGdump(ctx context.Context, cancel context.CancelFunc,
...
+			if err != nil {
+				return nil, fmt.Errorf("parse %s to int in line - %s: %v", setValArgsArr[1], sequencesTextFileData[i], err)
+			}
+			seqNameRaw := setValArgsArr[0][1 : len(setValArgsArr[0])-1]
+			seqName := sqlname.NewSourceNameFromQualifiedName(seqNameRaw)
+			sequenceValueMap[seqName] = seqVal
+		}
+	}
+	return sequenceValueMap, nil
+}
```
Comment (amit-yb) [inline yb-voyager/cmd/exportData.go:311]: Sometimes using smaller+idiomatic names can lead to more readable code. The name of the function itself provides some context for the names inside its scope. The context can be utilised to reduce the local name lengths.
```
func getPGDumpSequencesAndValues() (map[*sqlname.SourceName]int64, error) {
	result := map[*sqlname.SourceName]int64{}
	path := filepath.Join(exportDir, "data", "postdata.sql")
	bs, err := os.ReadFile(path)
	if err != nil {
		utils.ErrExit("read file %q: %v", path, err)
	}
	lines := strings.Split(string(bs), "\n")

    // Sample line: SELECT pg_catalog.setval('public.actor_actor_id_seq', 200, true);
	re := regexp.MustCompile(`(?i)SELECT.*setval\((?P<args>.*)\)`)

	for _, line := range lines {
		matches := re.FindStringSubmatch(line)
		if len(matches) == 0 {
			continue
		}
		idx := re.SubexpIndex("args")
		if idx > len(matches) {
			return nil, fmt.Errorf("invalid index %d for matches - %s for line %s", idx, matches, lines[i])
		}
		match := matches[idx]
		args := strings.Split(match, ",")

		seqNameRaw, _ := strconv.Unquote(args[0])
		seqName := sqlname.NewSourceNameFromQualifiedName(seqNameRaw)

		seqVal, err := strconv.ParseInt(strings.TrimSpace(args[1]), 10, 64)
		if err != nil {
			return nil, fmt.Errorf("parse %s to int in line - %s: %v", args[1], lines[i], err)
		}
		result[seqName] = seqVal
	}
	return result, nil
}
```

File: debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/SequenceObjectUpdater.java [hunk 2]
```diff
@@ -62,6 +60,27 @@ public void initColumnSequenceMap(String columnSequenceMapString){
             this.sequenceMax.put(sequenceName, this.sequenceMax.getOrDefault(sequenceName, (long)0));
         }
     }
+    public void initSequenceMax(String initSequenceMapString, ConcurrentMap<String, Long> sequenceMax){
+        this.sequenceMax = new ConcurrentHashMap<>();
```
Comment (amit-yb) [inline debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/SequenceObjectUpdater.java:64]: A comment showing sample contents of the `initSequenceMapString` would really help here.

**Review by amit-yb — APPROVED** [review]
LGTM

File: yb-voyager/cmd/exportData.go [hunk 3]
```diff
@@ -264,6 +277,39 @@ func exportPGSnapshotWithPGdump(ctx context.Context, cancel context.CancelFunc,
...
+	if err != nil {
+		utils.ErrExit("read file %q: %v", sequenceFilePath, err)
+	}
+
+	sequencesTextFileData := strings.Split(string(seqeuencesTextFileDataBytes), "\n")
+	numLines := len(sequencesTextFileData)
+	setvalRegex := regexp.MustCompile(`(?i)SELECT.*setval\((?P<args>.*)\)`)
+
+	for i := 0; i < numLines; i++ {
+		matches := setvalRegex.FindStringSubmatch(sequencesTextFileData[i])
```
Comment (sanyamsinghal) [inline yb-voyager/cmd/exportData.go:293]: a comment explaining the regexp matching etc with an example would help.

**Pull request merged**